python/semantic_kernel/connectors/ai/ollama/services/ollama_chat_completion.py

```diff
@@ -30,11 +30,12 @@ class OllamaChatCompletion(TextCompletionClientBase, ChatCompletionClientBase, A
     Make sure to have the ollama service running either locally or remotely.
 
     Arguments:
-        ai_model_id {str} -- Ollama model name, see https://ollama.ai/library
+        ai_model_id {str} -- Ollama model name, defaults to llama2 (the most popular model at https://ollama.ai/library)
         url {Optional[Union[str, HttpUrl]]} -- URL of the Ollama server, defaults to http://localhost:11434/api/chat
         session {Optional[aiohttp.ClientSession]} -- Optional client session to use for requests.
     """
 
+    ai_model_id: str = "llama2"
```

Comment:
Let's please remove this default `llama2` ai_model_id value in this file and the ollama text embedding file, and this should PR should be good to go. Thank you.